### DISCUSSION TITLE
Regular call day/time-slot

### DISCUSSION THREAD

**Discussion by FabrizioMoggio** [camaraproject/CallForwardingSignal#102] [General]
Considering:
- Confluence is setting up scheduled meeting in a regular basis (biweekly)
- The current timeslot and day is not right anymore for some partner
- I don't plan to have anyway such a tight scheduling 

I suggest to agree on a different day-time slot.

**Comment by FabrizioMoggio** [comment]
Proposal:  Monday 14-15 CET

**Comment by FabrizioMoggio** [comment]
Can we consider this time slot as agreed?

**Reply by StefanoFalsetto-CKHIOD** [reply]
It is OK for me.

**Reply by bigludo7** [reply]
Hello - works for me as well.

**Comment by FabrizioMoggio** [comment]
**Conclusion**: new regular meeting scheduling on Monday 14-15 CET

**Comment by FabrizioMoggio** [comment]
Considering: https://github.com/camaraproject/CallForwardingSignal/discussions/107

I reopen this discussion proposing to suspend the regular call until some of the actions listed in #107 provide us with some more topic to discuss.

**Reply by bigludo7** [reply]
Work for me !

**Comment by FabrizioMoggio** [comment]
I will suspend the regular call.
